index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <title>Grow Megha Portfolio</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="header">
+        <ul class="nav-items">
+          <li class="logo"><a>Megha</a></li>
+          <li class="menu"><i class="fa-solid fa-bars"></i></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="main-section">
+      <div class="FirstShape">
+        <img src="/images/Ellipse8.png" alt="shapes"/>
+      </div>
+      <div class="first">
+        <img class="half-ellipse" src="/images/Ellipse11.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="Ellipse5" src="/images/Ellipse5.png" alt="shapes"/>
+      </div>
```

Comment:
- [ ] The image should not be part of the content (aesthetic reasons) please use CSS background images instead.